app/routes/settings/users/index.js

```diff
@@ -42,7 +42,7 @@ export default class SettingsUsersIndexRoute extends Route {
         number: params.page,
         size: params.size,
       },
-      include: 'memberships.organization,memberships.role,memberships.status,status',
+      include: 'memberships.organization,memberships.role,memberships.status,status,login-activity',
```

Comment:
Kan je inschatten wat impact van die extra include is op performantie?
Misschien kan lijst ondertussen ook overzichtelijker geschreven worden met `[...].join(',')`?

Reply:
|  | Met `login-activity` | Zonder `login-activity`|
|---|---|---|
1st run | 7.80s | 6.36s |
2nd run | 6.97s | 7.12s |
3rd run | 7.14s | 6.59s |

Vrij gelijkaardige resultaten. Maar ik heb deze wel altijd uitgevoerd met 10 gebruikers per view, mogelijks beter om hier de login activiteit later in te laden.

Reply:
Ok om weg te laten voor mij. Ik beschouw dat toch eerder als side-data dan main data.